cmd/nerdctl/run_restart.go

```diff
@@ -0,0 +1,67 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/containerd/containerd"
+	"github.com/containerd/containerd/runtime/restart"
+	"github.com/containerd/nerdctl/pkg/strutil"
+)
+
+func generateRestartOpts(ctx context.Context, client *containerd.Client, restartFlag, logURI string) ([]containerd.NewContainerOpts, error) {
+	policySlice := strings.Split(restartFlag, ":")
+	switch policySlice[0] {
+	case "", "no":
```

Comment:
empty policy should be supported too ?
  https://github.com/containerd/containerd/blob/main/runtime/restart/restart.go#L85

Reply:
For containerd, empty policy is to be compatible with scenes without policy label.